docs/deployment/azure/local-provisioning.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Local Azure provisioning
+description: Learn how to use Azure resources in your local development environment.
+ms.date: 04/05/2024
+---
+
+# Local Azure provisioning
+
+.NET Aspire simplifies local cloud-native app development with its compelling app host model. This model allows you to run your app locally with the same configuration and services as in Azure.
+
+In this article you learn how to provision Azure resources from your local development environment through the [.NET Aspire app host](../../fundamentals/app-host-overview.md). All of this is possible with the help of the [Azure.Provisioning.* libraries](#azure-provisioning-libraries), which provide a set of APIs to provision Azure resources. These packages are transitive dependencies of the .NET Aspire Azure hosting libraries you use in your app host, so you don't need to install them separately.
+
+## Assumptions
+
+This article assumes that you have an Azure account and subscription. If you don't have an Azure account, you can create a free one at [Azure Free Account](https://azure.microsoft.com/free/). For provisioning functionality to work correctly, you'll need to be authenticated with Azure. Additionally, you'll need to provide some configuration values so that the provisioning logic can create resources on your behalf.
+
+## Configuration
+
+When utilizing Azure resources in your local development environment, you need to provide the necessary configuration values. Configuration values are specified under the `Azure` section:
+
+- `SubscriptionId`: The Azure subscription ID.
+- `AllowResourceGroupCreation`: A boolean value that indicates whether to create a new resource group.
+- `ResourceGroup`: The name of the resource group to use.
+- `Location`: The Azure region to use.
+- `Tenant`: The Azure tenant ID, used as the prefix to `<tenant-id>.onmicrosoft.com`. For more information, see [Quickstart: Set up a tenant](/entra/identity-platform/quickstart-create-new-tenant).
+
+Consider the following example _appsettings.json_ configuration:
+
+```json
+{
+  "Azure": {
+    "SubscriptionId": "<Your subscription id>",
+    "AllowResourceGroupCreation": true,
+    "ResourceGroup": "<Valid resource group name>",
+    "Location": "<Valid Azure location>",
+    "Tenant": "<Your tenant>.onmicrosoft.com"
+  }
+}
+```
+
+> [!IMPORTANT]
+> It's recommended to store these values as app secrets. For more information, see [Manage app secrets](/aspnet/core/security/app-secrets).
+
+After you've configured the necessary values, you can start provisioning Azure resources in your local development environment.
```

Comment:
instead of 'you can start...' is it more accurate to say ", .NET Aspire will start automatically provisioning dependent Azure resources for your local development environment."?